layout-server/src/test/java/org/finos/vuu/layoutserver/service/ApplicationLayoutServiceTest.java

```diff
@@ -75,7 +77,7 @@ public void createApplicationLayout_invalidDefinition_throwsJsonException() {
         String definition = "invalid JSON";
 
         assertThrows(JsonProcessingException.class, () ->
-            service.persistApplicationLayout("user", objectMapper.readTree(definition))
+            service.persistApplicationLayout("user", (ObjectNode) objectMapper.readTree(definition))
```

Comment:
We might not need to cast if we do this: `objectMapper.readValue(definition, ObjectNode.class)`. Alternatively, could we make use of `objectNodeConverter`?

Reply:
also cleaned this up :)